RELEASING.md

```diff
@@ -1,9 +1,10 @@
 ## Releasing
 
+1. Update the version number in the `akka-management-xx-stable` project name in the [whitesource web UI](https://saas.whitesourcesoftware.com)
```

Comment:
thanks, can you add same thing in the other repos

Reply:
I think I covered them all now 👍